### PR TITLE
v1.4.0

### PR DESCRIPTION
**Pull request by jessgusclark**
New version which includes:

- Fix Don't show Again #279
- Hide Nifty Wallet if the user doesn't have it #286

### CODE REVIEW

**Comment by jessgusclark** [comment]
Waiting until Monday to merge this. I don't want to release a new version on a Friday ;-)

**Review by lucachaco — APPROVED** [review]

**Pull request merged**
